sstable/src/value/range.rs

```diff
@@ -0,0 +1,95 @@
+use std::io;
+use std::ops::Range;
+
+use crate::value::{deserialize_u64, ValueReader, ValueWriter};
+
+#[derive(Default)]
+pub struct RangeReader {
```

Comment:
are comment to describe what this does and what it's used for would be good

Reply:
added comment